.editorconfig

```diff
@@ -3,7 +3,7 @@ root = true
 [*]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```

Comment:
When I do 

```shell
npx eslint --print-config src/App.vue  |grep -A 3 indent
    "indent": [
      "error",
      2,
      {
```
I see the indent of eslint is configured to 2.

Can you add a overwrite in .eslintrc.js to make it 4?

Reply:
maybe use https://www.npmjs.com/package/eslint-plugin-editorconfig